src/terminal.go

```diff
@@ -1758,6 +1759,46 @@ func (t *Terminal) executeCommand(template string, forcePlus bool, background bo
 	cleanTemporaryFiles()
 }
 
+func (t *Terminal) executeChangeQuery(template string) {
+	valid, list := t.buildPlusList(template, false)
+	if !valid {
+		return
+	}
+	command := t.replacePlaceholder(template, false, string(t.input), list)
+	cmd := util.ExecCommand(command, false)
+	newQuery := []rune{}
+	t.executing.Set(true)
+	cmd.Stdin = tui.TtyIn()
+	out, _ := cmd.StdoutPipe()
+	t.tui.Pause(false)
+	cmd.Stderr = cmd.Stdout
+	reader := bufio.NewReader(out)
+	err := cmd.Start()
+	if err != nil {
+		newQuery = append(newQuery, []rune(err.Error())...)
+	} else {
+		for {
+			line, err := reader.ReadString('\n')
+			newQuery = append(newQuery, []rune(line)...)
+			if err != nil {
+				break
+			}
+		}
+	}
+	err = cmd.Wait()
+	if err != nil {
+		newQuery = append(newQuery, []rune(err.Error())...)
+	}
```

Comment:
I based this off of a combination of what's done for the preview pane and `executeCommand`. I'm not too familiar with the behavior of either the `exec` or `bufio` libraries, though, so I'm not sure if there's a better way.

I initially tried `cmd.Run()` but I must have been missing something because I couldn't figure out how to get any output from it.

I didn't use `cmd. CombinedOutput()` because it returns a byte array, and I didn't want to have to try to correctly convert that to a rune array (needed for `t.input`). But converting strings to rune arrays seems straight-forward.